features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -162,14 +162,21 @@ export function SidebarNavigation() {
     <Container isCollapsed={isSidebarCollapsed}>
       <FixedContainer>
         <Header>
-          <Logo
-            src={
-              isSidebarCollapsed
-                ? "/icons/logo-small.svg"
-                : "/icons/logo-large.svg"
-            }
-            alt="logo"
-          />
+          <Logo>
+            <source
```

Comment:
Very nice solution indeed. Had to read up on the `<source>` tag. Now I got it. Basically, `<source>` is used if there are matches (always large logo for screens smaller than desktop) otherwise `<img>` is used. Right?

I'm planning to create a list of alternative solutions and this will be one of them 😄

Reply:
Essentially yeah, the `<img>` tag is still always in use no matter what, but the `src` attribute on the `<img>` is switched for whatever is listed in the `<source>` when the media query is true. That way we still get to use all the same styling and alt text on the original `<img>`.

I'm not sure if the same could have been achieved using a plain `<img>` in combination with the `srcset` and `sizes` attributes, but I couldn't quite get that working myself.